src/UIKit/UIContentSizeCategory.cs

```diff
@@ -0,0 +1,38 @@
+﻿#if !WATCH
+
+using System;
+using System.Runtime.InteropServices;
+
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+
+namespace XamCore.UIKit {
+	static public partial class UIContentSizeCategoryExtensions {
+
+		[DllImport (Constants.UIKitLibrary)]
+		static extern nint /* NSComparisonResult */ UIContentSizeCategoryCompareToCategory (IntPtr /* NSString */ lhs, IntPtr /* NSString */ rhs);
+
+		[iOS (11, 0), TV (11, 0)]
+		public static NSComparisonResult Compare (UIContentSizeCategory category1, UIContentSizeCategory category2)
+		{
+			if (!UIContentSizeCategory.IsDefined (typeof (UIContentSizeCategory), category1))
+				throw new ArgumentException ($"Unknown 'UIContentSizeCategory' value", nameof (category1));
```

Comment:
that's a costly check - any reason for it ?

Reply:
Guess this comment is being debated in Rolf's comment above